src/components/DropdownFilterModal/DropdownFilterInput.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, FunctionComponent } from "react";
+import { StyleSheet, View, TextInput } from "react-native";
+import { size, borderRadius, color, fontSize } from "../../common/styles";
+import { AppText } from "../Layout/AppText";
+import { DropdownFilterModal, DropdownItem } from "./DropdownFilterModal";
+
+const styles = StyleSheet.create({
+  label: {
+    fontFamily: "brand-bold"
+  },
+  inputView: {
+    marginTop: size(1),
+    minHeight: size(6),
+    borderRadius: borderRadius(2),
+    borderWidth: 1,
+    backgroundColor: color("grey", 0),
+    borderColor: color("blue", 50)
+  },
+  inputText: {
```

Comment:
Need to add `fontFamily: brand-regular` too, right now it's the default font

Reply:
ok done.